### PR TITLE
Fix document using DRI tickets

### PR DESCRIPTION
**Pull request by hzy46**
To address #4760

Review:
- [x] Recommended Practices
- [x] Installation Guide
- [x] Pinned Cell
- [x] storage-manager
- [x] Alerting and Troubleshooting

### CODE REVIEW

File: docs/manual/cluster-admin/alerting-and-troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  smtp_from: <smtp-email-address>
+  smtp_url: <smtp-server>:<smtp-port>
+```
+
+Configuration `port` stands for the port of alert manager. In most cases, you don't need to change it. Configuration `receiver` is usually set to be the administrator's e-mail address to receive alerting e-mails.
+
+Save the configuration file, and start alert manager by:
+
+```bash
+./paictl.py service start -n alert-manager
```
Comment (Binyang2014) [inline docs/manual/cluster-admin/alerting-and-troubleshooting.md:32]: `./paictl.py service stop -n alert-manager` ?

File: docs/manual/cluster-admin/alerting-and-troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+## Troubleshooting
+
+### PaiServicePodNotReady Alert
+
+This is a kind of alert from alert manager, and usually caused by container being killed by operator or OOM killer. To check if it was killed by OOM killer, you can check node's free memory via Prometheus:
+
+  1. visit Prometheus web page, it is usually `http://<your-pai-master-ip>:9091`.
+  2. Enter query `node_memory_MemFree_bytes`.
+  3. If free memory drop to near 0, the container should be killed by OOM killer
```
Comment (Binyang2014) [inline docs/manual/cluster-admin/alerting-and-troubleshooting.md:50]: I think the most OOM cases not caused by service exhausted host memory. But PAI service try to allocate memory exceed pod memory limit. The better way to check the OOM issue is run  `docker inspect <stop_docker_id>`. There is a OOM filed.

`dmesg`  also a good way

File: docs/manual/cluster-admin/alerting-and-troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+  4. You can double check this by logging into node and run command `dmesg` and looking for phase `oom`.
+
+Solutions:
+
+  1. Force remove unhealth containers with this command in terminal:
+  `kubectl delete pod pod-name --grace-period=0 --force`
+  2. Recreate pod in Kubernetes, this operation may block indefinitely because dockerd may not functioning correctly after OOM. If recreate blocked too long, you can log into the node and restart dockerd via `/etc/init.d/docker restart`.
```
Comment (Binyang2014) [inline docs/manual/cluster-admin/alerting-and-troubleshooting.md:57]: Should we increase pod memory limit for this case?

File: docs/manual/cluster-admin/alerting-and-troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+If the node's ready label has value "unknown", this means the node may disconnect from Kubernetes master, this may due to several reasons:
+
+  - Node is down
+  - Kubelet is down
+  - Network partition between node and Kubernetes master
+
+You can first try to log into the node. If you can not, and have no ping response, the node may be down, and you should boot it up.
+
+If you can log in to the node, you should check if the kubelet is up and running, execute `sudo docker ps` command in the node, normally the kubelet container will be listed. The output should like:
```
Comment (Binyang2014) [inline docs/manual/cluster-admin/alerting-and-troubleshooting.md:77]: After using kubespray, there is no container named kubelet. Kubelet run as host service. Should checked by `sudo systemctl status kubelet`

File: docs/manual/cluster-admin/alerting-and-troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+After this step, you should check the log of kubelet, to see if it can access Kubernetes API. If you see something like:
+
+```
+  E0410 04:24:30.663050    2491 kubelet_node_status.go:386] Error updating node status, will retry: error getting node "10.0.0.1": Get http://10.0.1.2:8080/api/v1/nodes/10.0.0.1: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
+```
+
+This means the node can not report its status to Kubernetes, and hence the Kubernetes will post unknown status, and this triggered the alert.
+
+You should check what caused this connectivity problem.
```
Comment (Binyang2014) [inline docs/manual/cluster-admin/alerting-and-troubleshooting.md:91]: We should replace this to new version. This is for PAI which version is below 0.16.x

File: docs/manual/cluster-admin/alerting-and-troubleshooting.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+### NodeFilesystemUsage Alert
+
+This is a kind of alert from alert manager, and is used to monitor disk space of each server. If usage of disk space is greater than 80%, this alert will be triggered. OpenPAI has two services may use a lot of disk space. They are storage manager and docker image cache. If there is other usage of OpenPAI servers, they should be checked to avoid the disk usage is caused by outside of OpenPAI.
+
+Solutions:
+
+  1. Check user file on the NFS storage server launched by storage manager. If you didn't set up a storage manager, ignore this step.
+  2. Check the docker cache. The docker may use too many disk space for caching, it's worth to have a check.
```
Comment (Binyang2014) [inline docs/manual/cluster-admin/alerting-and-troubleshooting.md:100]: Also should check PAI log folder size. The path is `/var/log/pai`

**Review by abuccts — APPROVED** [review]

File: docs/manual/cluster-admin/alerting-and-troubleshooting.md (hunk 2 above)
Comment (hzy46) [reply docs/manual/cluster-admin/alerting-and-troubleshooting.md:50]: Added suggestion: Or you can run `docker inspect <stopped_docker_id>` to get more detailed information.

File: docs/manual/cluster-admin/alerting-and-troubleshooting.md (hunk 3 above)
Comment (hzy46) [reply docs/manual/cluster-admin/alerting-and-troubleshooting.md:57]: Added the following:

```
3. If restarting doesn't solve it, you can consider increase the pod's memory limit.
```

File: docs/manual/cluster-admin/alerting-and-troubleshooting.md (hunk 4 above)
Comment (hzy46) [reply docs/manual/cluster-admin/alerting-and-troubleshooting.md:77]: Done

File: docs/manual/cluster-admin/alerting-and-troubleshooting.md (hunk 5 above)
Comment (hzy46) [reply docs/manual/cluster-admin/alerting-and-troubleshooting.md:91]: As far as I'm concerned, the kubelet's log should be similar?

File: docs/manual/cluster-admin/alerting-and-troubleshooting.md (hunk 6 above)
Comment (hzy46) [reply docs/manual/cluster-admin/alerting-and-troubleshooting.md:100]: Added the following:

```
3. Check PAI log folder size. The path is `/var/log/pai`.
```

File: docs/manual/cluster-admin/alerting-and-troubleshooting.md (hunk 1 above)
Comment (hzy46) [reply docs/manual/cluster-admin/alerting-and-troubleshooting.md]: Done.

**Review by Binyang2014 — APPROVED** [review]

**Review by scarlett2018 — APPROVED** [review]

**Pull request merged**